editor/plugins/sprite_2d_editor_plugin.cpp

```diff
@@ -665,8 +665,17 @@ Sprite2DEditor::Sprite2DEditor() {
 	add_child(debug_uv_dialog);
 }
 
+void Sprite2DEditorPlugin::_editor_theme_changed() {
```

Comment:
Looks like this method should be called somewhere? (other than constructor)
Sprite2DEditor is a Control, so you can just use NOTIFICATION_THEME_CHANGED.

Reply:
Then you can connect `theme_changed` from Sprite2DEditor to this method. It needs to be called with each theme update.